tests/ui/const-generics/issue-112505-overflow.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0512]: cannot transmute between types of different sizes, or dependently-sized types
+  --> $DIR/issue-112505-overflow.rs:4:14
+   |
+LL |     unsafe { std::mem::transmute(v) }
+   |              ^^^^^^^^^^^^^^^^^^^
+   |
+   = note: source type: `[[[u32; 8888888]; 9999999]; 777777777]` (values of the type `[[[u32; 8888888]; 9999999]; 777777777]` are too big for the current architecture)
+   = note: target type: `[[[u32; 9999999]; 777777777]; 239]` (59484438436515561504 bits)
```

Comment:
could you also test the `xxx bytes` message?

Reply:
we can not construct this test case, we will report `are too big for the current architecture` for larger arrays.

`u128` is definitely enough for this, but I don't want to use `unwrap`...
any better idea?

Reply:
If it is expected to not overflow, use the bug macro.

Reply:
This is better?
```rust
u128::from(size.bytes()).checked_mul(8).map_or("", |v| format!("{} bits", v))
```

Reply:
> If it is expected to not overflow, use the bug macro.

sounds good!

Reply:
Why? If you are *absolutely* sure that multiplying by 8 does not overflow, use `bug!`. If you are not sure, then see if you can find a test case.

Reply:
fixed.